packages/cli/README.md

```diff
@@ -30,7 +30,7 @@ USAGE
 
 ## 🔑 Authentication
 
-The last version of Google Spreadsheets API requires us to be authenticated to allow fetching spreadsheet data. Read more about [authentication here](http://ackeecz.github.io/lokse/authentication)
+The last version of Google Spreadsheets API requires us to be authenticated to allow fetching spreadsheet data. Read more about [authentication here](https://github.com/AckeeCZ/lokse/blob/master/docs/web/src/pages/en/authentication.md)
```

Comment:
you're right, the URL is broken, but the correct one is `https://ackeecz.github.io/lokse/en/authentication/`